Box.V2/Models/BoxTaskAssignment.cs

```diff
@@ -57,9 +58,25 @@ public class BoxTaskAssignment : BoxEntity
         /// <summary>
         /// Gets the state of the resolution.
         /// </summary>
+        [Obsolete("This field is deprecated, and may not work consistently.  Use Status or StatusTranslated instead.")]
+        public ResolutionStateType? ResolutionState {
+            get
+            {
+                return (ResolutionStateType) System.Enum.Parse(typeof(ResolutionStateType), Status, ignoreCase: true);
+            }
+        }
+
+        /// <summary>
+        /// Gets the resolution status of the task assignment.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldStatus)]
+        public string Status { get; private set; }
+
+        /// <summary>
+        /// Gets the translated/human-readable resolution status of the task assignment.
+        /// </summary>
         [JsonProperty(PropertyName = FieldResolutionState)]
-        [JsonConverter(typeof(StringEnumConverter))]
-        public ResolutionStateType? ResolutionState { get; private set; }
+        public string StatusTranslated { get; private set; }
```

Comment:
Use "Localization" instead of "Translation" to sound more technical, so either `StatusLocalized` or `LocalizedStatus`. Personally, I would like the new fields to be `CanonicalStatus` and `LocalizedStatus`, but I can see the advantage of having API response fields consistently map to SDK object fields (ie I'm fine with `Status` and `LocalizedStatus`).